sql/testdata/builtin_function

```diff
@@ -22,7 +22,7 @@ SELECT LOWER('RoacH7')
 LOWER('RoacH7')
 roach7
 
-statement error LOWER: argument type mismatch: string expected, but found int
+statement error unknown signature for LOWER: LOWER\(int\)
 SELECT LOWER(32)
 
 query T
```

Comment:
added random but not concat since we currently can't test for the empty string :(